src/components/plots/linePlot.ts

```diff
@@ -27,6 +27,16 @@ export module Plot {
       this._defaultStrokeColor = new Scale.Color().range()[0];
     }
 
+    public interpolate(): string | ((points: number[][]) => string);
+    public interpolate(interpolationMode: string | ((points: number[][]) => string)): void;
+    public interpolate(interpolationMode?: any) {
+      if (interpolationMode === null) {
+        return this._interpolationMode;
+      } else {
+        this._interpolationMode = interpolationMode;
+      }
```

Comment:
return this;  Also, the else statement is not necessary / it makes for unecessarily nested code